tests/std/tests/Dev11_0836436_get_time/test.cpp

```diff
@@ -208,5 +250,297 @@ void test_990695() {
             assert(t.tm_mday == 31);
             assert(t.tm_year == 114);
         }
+
+        {
+            istringstream iss("sep 31 2014");
+            ios_base::iostate err = Bit;
+            tm t{};
+            const string fmt("%b %d %Y");
+            use_facet<time_get<char>>(iss.getloc())
+                .get(Iter(iss.rdbuf()), Iter(), iss, err, &t, fmt.c_str(), fmt.c_str() + fmt.size());
+            assert(t.tm_mon == 8);
+            assert(t.tm_mday == 31);
+            assert(t.tm_year == 114);
+        }
+
+        {
+            istringstream iss("aUG 14 2020");
+            ios_base::iostate err = Bit;
+            tm t{};
+            const string fmt("%b %d %Y");
+            use_facet<time_get<char>>(iss.getloc())
+                .get(Iter(iss.rdbuf()), Iter(), iss, err, &t, fmt.c_str(), fmt.c_str() + fmt.size());
+            assert(t.tm_mon == 7);
+            assert(t.tm_mday == 14);
+            assert(t.tm_year == 120);
+        }
+
+        {
+            istringstream iss("feBRuArY 02 1991");
+            ios_base::iostate err = Bit;
+            tm t{};
+            const string fmt("%b %d %Y");
+            use_facet<time_get<char>>(iss.getloc())
+                .get(Iter(iss.rdbuf()), Iter(), iss, err, &t, fmt.c_str(), fmt.c_str() + fmt.size());
+            assert(t.tm_mon == 1);
+            assert(t.tm_mday == 2);
+            assert(t.tm_year == 91);
+        }
+
+        {
+            istringstream iss("19 SKIP_THIS sEpTemBER SKIP_THIS 2005");
+            ios_base::iostate err = Bit;
+            tm t{};
+            const string fmt("%d SKIP_THIS %b SKIP_THIS %Y");
+            use_facet<time_get<char>>(iss.getloc())
+                .get(Iter(iss.rdbuf()), Iter(), iss, err, &t, fmt.c_str(), fmt.c_str() + fmt.size());
+            assert(t.tm_mon == 8);
+            assert(t.tm_mday == 19);
+            assert(t.tm_year == 105);
+        }
+
+        {
+            // This case should fail
+            istringstream iss("2011-D-18");
+            ios_base::iostate err = Bit;
+            tm t{};
+            const string fmt("%Y-%b-%d");
+            use_facet<time_get<char>>(iss.getloc())
+                .get(Iter(iss.rdbuf()), Iter(), iss, err, &t, fmt.c_str(), fmt.c_str() + fmt.size());
+            assert(err == ios_base::failbit);
+        }
+
+        {
+            std::tm t = {};
+            std::istringstream ss("2018-M-18");
+            ss >> std::get_time(&t, "%Y-%b-%d");
+            assert(ss.fail());
+        }
     }
 }
+
+void test_locale_russian() {
+    // Russian January in different cases (short, long, mixed cases)
+    assert(read_date_locale(L"2020-\x042f\x043d\x0432\x0430\x0440\x044c-05", "ru_RU.UTF-8")
+           == make_tuple(5, /*NOTE DIFFERENCE:*/ 0, 120));

```

Comment:
> I'm unsure of the intent of the `/*NOTE DIFFERENCE:*/` comments. Since they occur in every single call to `make_tuple`, it seems that the same sentiment could be conveyed by a simple, overarching comment at the top of the method perhaps. However, if the intent is just "note the difference in what is being tested in each assert", I think the comment may be altogether unnecessary since (1) the differences can be observed by reading the code and (2) with multiple asserts in a single function, I think they would be assumed to be different, as they would otherwise be redundant.

I agree. These comments are redundant.  They just emphasize that month numbers returned by the `read_date_locale`  function starting from 0, which I think is quite obvious in the function body. However, I tried to follow the pattern of lines 145 and 147. I thought it is helpful for some unknown reasons!

I removed them completely in my test cases.

Reply:
Ah, I didn't notice the pre-existing pattern while reviewing.  In general, following the style and conventions of surrounding code is a good practice!  However, in this case it seems to instead highlight this potentially useless pattern in these tests.  
I suppose those occurrences might make some sense because of the year difference (77 as year yields 77 in the year field, whereas 11 as year yields 111 in the year field) but I think especially in these new test cases they don't help much.  There could be an argument made for removing the pre-existing occurrences, but I don't feel strongly enough to touch that code.  

Thanks for explaining the rationale and removing these occurrences!